Add new 6.6.2 capabilities, notations, and scriptlets

jon-strabala opened the pull request:
This is my first pass at adding the 6.6.2 changes to the Eventing documents
I could add pictures if needed, but I think it should be good at this point.

jon-strabala committed:
create new Scriptlet simpleTimer

jon-strabala committed:
Add simpleTimer to scriptlet index

jon-strabala committed:
Add Eventing scriptlet simpleTimer to main index

jon-strabala committed:
Add new scriptlet convertXMLtoJSON

jon-strabala committed:
Add link to Eventing scriptlet convertXMLtoJSON

jon-strabala committed:
Add Eventing scriptlet convertXMLtoJSON to main index

jon-strabala committed:
minor typo

jon-strabala committed:
Add link for testBinaryVsJSON

jon-strabala committed:
Add new scriptlet testBinaryVsJSON

jon-strabala committed:
minor cleanup

jon-strabala committed:
rename document and scriptlet

jon-strabala committed:
discuss return value

jon-strabala committed:
Add scriptlet basicBinaryKV

jon-strabala committed:
update index for new scriptlets

jon-strabala committed:
Add new Eventing Scriptlets

basicBinaryKV
advancedBinaryKV

jon-strabala committed:
cleanup / typo

jon-strabala committed:
remove uneeded property

jon-strabala committed:
fix typo in key

jon-strabala committed:
Explain new binary doc support

jon-strabala committed:
better function more information

jon-strabala committed:
formatting tabs

jon-strabala committed:
clarity

jon-strabala committed:
Update eventing-faq.adoc

jon-strabala committed:
Add 6.6.2 notes and misc. updates

Explain 6.6.2 change in Feed Boundary
Gentle introduction of switch from Metadata Bucket to Eventing Storage for the required Bucket for Eventing.
Define Function Name
Define Function Description
Define Description
Add Settings (Feed Boundary, Context Size)

jon-strabala committed:
Improve overview / capabilities, add 6.6.2 update

jon-strabala committed:
detail “language compatibility” and binary doc suppression

jon-strabala committed:
More info on Binary docs

jon-strabala committed:
Improve “language compatibility” binary text

jon-strabala committed:
small clarification

jon-strabala committed:
More info on timers and also 6.6.2 updates

jon-strabala committed:
Identify timer_context_size

jon-strabala committed:
diction

timer context size may be adjusted up or down on a per Function handler basis

jon-strabala committed:
clarity and also Eventing Quota

Clarify the binary v. non-binary
Add note about memory Quota for Eventing
Gently introduce term that Metadata Bucket name change come 7.X as the Eventing Storage bucket
FAQ entry for stalls at 48+ workers, with workaround

jon-strabala committed:
change to data_type to datatype

Since Eventing 6.6.1 suppressed all binary docs 
a) input mutations via OnUpdate, 
b) basic bucket ops like GET via the exposed JavaScript map, i.e. var somedoc = src_bkt["someid"];
c) advanced assessors.

This regression is a name change of a field that couldn't really be used for anything.   It was only documented because it "leaked" into results for test examples.

In 6.6.2 Eventing can process binary functions at this "language compatibility" mode, but not at any prior modes.

jon-strabala committed:
Introduce binary documents

jon-strabala committed:
formatting

jon-strabala committed:
Add 6.6.2 updates

jon-strabala committed:
Updates for 6.6.2

jon-strabala committed:
6.6.2 update

jon-strabala committed:
Update for 6.6.2

jon-strabala committed:
Update for 6.6.2

jon-strabala committed:
Update fro 6.6.2

jon-strabala committed:
UPdate for 6.6.2

jon-strabala committed:
Update for 6.6.2

jon-strabala committed:
Update 6.6.2

jon-strabala committed:
Update 6.6.2

jon-strabala committed:
Update for 6.6.2

jon-strabala committed:
Update for 6.6.2

amarantha-k reviewed (approved):
Approving after a quick look. Will run a editorial review before merging.

jon-strabala committed:
Function worked on 7.0.0 bot on 6.6.X

Update to v8 runner added a new useful function, needed to code a similar for prior releases.

jon-strabala committed:
Test of cell spanning to improve layout

jon-strabala committed:
Merge pull request #1 from jon-strabala/Eventing-Rest-API-cell-spanning

Test of cell spanning to improve layout

jon-strabala committed:
make Note span 2-cols, space better

jon-strabala committed:
test of grid on/off

jon-strabala committed:
undo grid none experiment
